src/app/successmessage/successmessage.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+class success{
+success:string;
+
+constructor(success:string){
+
+  this.success = success;
+}
+
+}
+
+@Component({
+  selector: 'successmessage',
+  templateUrl: './successmessage.component.html',
+  template:`
```

Comment:
you cannot have both templateUrl and template defined for the meta-data here. you need to mention only one